frontend/src/metabase/querying/components/FilterModal/CoordinateFilterEditor/CoordinateFilterEditor.tsx

```diff
@@ -85,23 +84,26 @@ export function CoordinateFilterEditor({
 }
 
 interface NumberValueInputProps {
-  column: Lib.ColumnMetadata;
   values: NumberValue[];
   valueCount: number;
   hasMultipleValues?: boolean;
   onChange: (values: NumberValue[]) => void;
 }
 
 function NumberValueInput({
-  column,
   values,
   valueCount,
   hasMultipleValues,
   onChange,
 }: NumberValueInputProps) {
   if (hasMultipleValues) {
     return (
-      <FilterValuePicker value={values} column={column} onChange={onChange} />
+      <FilterValuePicker
```

Comment:
Coordinates used to be formatted like this:

![image](https://github.com/metabase/metabase/assets/6830683/3df9dca0-62d6-4a80-b5df-6dfcbcfc4fc0)

Now they're not.
Is this going to be handled in a separate PR?

Reply:
https://github.com/metabase/metabase/issues/36749